### PR TITLE
[Advanced Article] Add page for Lava ipRPC 

### PR DESCRIPTION
**Pull request by KagemniKarimu**
This PR includes a new Advanced Tutorial introducing Lava and explaining how to use the newly available Lava ipRPC endpoints.

### CODE REVIEW

File: docs/articles/advanced/using-iprpc.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: 11
+---
+
+# Using Evmos ipRPC
+
+## Overview  📋
```
Comment (danburck) [inline docs/articles/advanced/using-iprpc.md:7]: nit 
Add empty line after titles (applies to whole file)

```suggestion
## Overview  📋

```

File: docs/articles/advanced/using-iprpc.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+- **Tendermint-RPC** - `https://tm.evmos-testnet.lava.build`
+- **Tendermint-RPC WSS** - `wss://tm.evmos-testnet.lava.build/websocket`
+- **gRPC** - `grpc.evmos-testnet.lava.build:443`
+- **Web-gRPC** - `https://grpc.evmos-testnet.lava.build`
+
+## Setting your `evmosd` to use ipRPC by default ⚡
+
+You can set your `evmosd` installation to use ipRPC for all calls and requests. To do so, you need to configure your default node to use the Tendermint-RPC URL for your respective network.
+
+⚆ For `mainnet` use:
```
Comment (danburck) [inline docs/articles/advanced/using-iprpc.md:41]:
```suggestion
For `mainnet` use:
```

File: docs/articles/advanced/using-iprpc.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+```bash
+./evmosd config node https://tm.evmos.lava.build:443
+```
+
+⚆ For `testnet` use:
```
Comment (danburck) [inline docs/articles/advanced/using-iprpc.md:46]:
```suggestion
For `testnet` use:
```

File: docs/articles/advanced/using-iprpc.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+```bash
+./evmosd config node https://tm.evmos-testnet.lava.build:443
+```
+
+⚆ To ensure that your `evmosd` is correctly setup run the following:
```
Comment (danburck) [inline docs/articles/advanced/using-iprpc.md:51]:
```suggestion
To ensure that your `evmosd` is correctly setup run the following:
```

File: docs/articles/advanced/using-iprpc.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+```bash
+./evmosd status 
+```
+
+🗸 If you get a readout like the following, you're properly configured!
```
Comment (danburck) [inline docs/articles/advanced/using-iprpc.md:56]:
```suggestion
If you get a readout like the following, you're properly configured!
```

**Review by danburck — APPROVED** [review]
Thanks @KagemniKarimu 

I left some nit comments, can you address?

**Comment by KagemniKarimu** [comment]
@danburck 
All of your suggested changes have been committed! :100: 
Thank you for your review!

**Pull request merged**